### DISCUSSION TITLE
Relationships 1:n

### DISCUSSION THREAD

**Discussion by mauricioranzan** [opengisch/QField#2868] [Q&A]
Hi,
My name is Mauricio.
I'm Brazilian and I don't speak English, so I apologize for any mistakes.

I created a project in QGis with some 1:n relationships that worked in QGis.
In QField I can also do the records correctly, but when viewing a table of related data, all records appear as "0" (zero) in the list. But, if I open the registry, the data is there.
The problem is only in the list of records.
Relationships 1:n

![qfield](https://user-images.githubusercontent.com/105087582/167230224-878a795a-945e-4d6d-843e-cc9e89c2054c.jpg)

**Comment by suricactus** [comment]
What you see there is controlled with the display expression on your layer. You can change it to whatever you want from here:
![image](https://user-images.githubusercontent.com/2820439/167243356-934dbe9c-581f-44aa-bc0b-5c8a09fd8c9d.png)
